docs/user-guide/deploy-chart/examples/deploying-kube-prometheus-helm-chart.md

```diff
@@ -0,0 +1,55 @@
+# Deploying Kube-Prometheus Helm Chart
+
+## Introduction
+
+`bitnami/kube-prometheus` Helm chart bootstraps a single node kube-prometheus deployment on a Kubernetes cluster using the Helm package manager.
+
+![](../../../images/deploy-chart/examples/kube-prometheus/kube-prometheus.png)
+
+## 1. Discover Kube-Prometheus chart from Chart Store
+
+Select `Charts` from the left panel to visit the `Chart Store` page. You will see numerous of charts on the page from which you have to find `bitnami/kube-prometheus` chart. You also can use the search bar to search the MySQL chart.
```

Comment:
Select `Chart Store` from the left panel. On the `Search charts` bar, type `kube-prometheus`, click `bitnami/kube-prometheus`.